### ISSUE TITLE
shorthand_operator: false positive: `seconds = seconds * 60 + value`

### ISSUE THREAD

**Issue by dusek** [realm/SwiftLint#1254]
`seconds = seconds * 60 + value`

We don't want to rewrite it to

```
seconds *= 60
seconds += value
```

as that is simply 2 lines and somehow destroys the code and mind flow.

Swiflint: 0.16.1

**Comment by marcelofabri** [comment]
Are you sure you're using `0.16.1`? This should be fixed on it (and I couldn't reproduce it).

**Comment by dusek** [comment]
@marcelofabri I am using 0.16.1. However I made mistake in the repro in this issue, I am very sorry. My original comment is now updated with the correct repro.

**Comment by marcelofabri** [comment]
@dusek Thanks for updating, I could reproduce it now. 

For now, you can either wrap the multiplication in parentheses (`seconds = (seconds * 60) + value`) or change the order of operands (`seconds = 60 * seconds + value`) to prevent this false positive.

**Comment by farzadshbfn** [comment]
using `0.18.1` and this problem still exists.
